content/docs/cml-with-dvc.md

```diff
@@ -72,8 +72,8 @@ jobs:
           cml comment create report.md
 ```
 
-See the [example repository](https://github.com/iterative/cml_dvc_case) for
-more, or check out the
+See the [example repository](https://github.com/iterative-test/cml-example-dvc)
+for more, or check out the
```

Comment:
Hi. Is `cml_dvc_case` going away? Just curious

Also I noticed it's currently mentioned in /src/components/pages/Home/UseCasesSection/index.tsx

Reply:
> _Is `cml_dvc_case` going away?_

Has been renamed (moved) but remains available at [`cml-example-dvc`](https://github.com/iterative-test/cml-example-dvc)

Reply:
> _Also I noticed it's currently mentioned in /src/components/pages/Home/UseCasesSection/index.tsx_

Where? 🤔

Reply:
Lines [180](https://github.com/iterative/cml.dev/blob/master/src/components/pages/Home/UseCasesSection/index.tsx#L180) and [225](https://github.com/iterative/cml.dev/blob/master/src/components/pages/Home/UseCasesSection/index.tsx#L225).

Reply:
@jorgeorpinel you are pointing master files, we have updated them in this PR

Reply:
@DavidGOrtega it's still there. Line [245](https://github.com/iterative/cml.dev/pull/362/files#diff-fee623e5534ec72a7f170c8e700632c061b40ba14c3cc8b7d15077e2d32ca58eR245).

Reply:
@jorgeorpinel, I can't see any link on the line you mention. Can you please copy a _permalink_ to the line? i.e. not on the review page, but on the file preview page.

Reply:
I did paste the permalink above. The diff for src/components/pages/Home/UseCasesSection/index.tsx doesn't load automatically though (too many changes) so it doesn't work well. Load the diff and find `cml-example-dvc. Idk if this is important BTW, just pointing it out

Reply:
Still can't find it. 🤯 Can you please use [this link](https://github.com/iterative/cml.dev/blob/f2cc1d2b15a7b09aca86d78b883006983f4ec3a1/src/components/pages/Home/UseCasesSection/index.tsx), click on the line number then on the three dots menu, and copy the permalink?

Reply:
Never mind, I no longer see `cml-example-dvc` anywhere. My bad

Reply:
Case closed 😅